diff --git a/helmfile/apps/keycloak/keycloak.yaml.gotmpl b/helmfile/apps/keycloak/keycloak.yaml.gotmpl
index 5e284cb..4235c48 100644
--- a/helmfile/apps/keycloak/keycloak.yaml.gotmpl
+++ b/helmfile/apps/keycloak/keycloak.yaml.gotmpl
@@ -176,6 +176,9 @@ keycloakConfigCli:
     master.json: |
       {
         "realm" : "master",
+        "displayName": "Open Suite Admin",
+        "displayNameHtml": "<b>Open Suite Admin</b>",
+        "loginTheme": "opensuite",
         "attributes": {
           "frontendUrl": "https://{{ .Values.global.hostname.keycloak }}.{{ .Values.global.domain }}"
         }
@@ -183,8 +186,9 @@ keycloakConfigCli:
     mijnbureau.json: |
       {
         "realm" : "mijnbureau",
-        "displayName": "Mijn Bureau",
-        "displayNameHtml": "<b>Mijn Bureau</b>",
+        "displayName": "Open Suite",
+        "displayNameHtml": "<b>Open Suite</b>",
+        "loginTheme": "opensuite",
         "defaultSignatureAlgorithm": "RS256",
         "enabled" : true,
         "revokeRefreshToken": true,
