diff --git a/helmfile/apps/keycloak/keycloak.yaml.gotmpl b/helmfile/apps/keycloak/keycloak.yaml.gotmpl
index 5e284cb..4235c48 100644
--- a/helmfile/apps/keycloak/keycloak.yaml.gotmpl
+++ b/helmfile/apps/keycloak/keycloak.yaml.gotmpl
@@ -183,8 +183,8 @@ keycloakConfigCli:
     mijnbureau.json: |
       {
         "realm" : "mijnbureau",
-        "displayName": "Mijn Bureau",
-        "displayNameHtml": "<b>Mijn Bureau</b>",
+        "displayName": "Open Suite",
+        "displayNameHtml": "<b>Open Suite</b>",
         "defaultSignatureAlgorithm": "RS256",
         "enabled" : true,
         "revokeRefreshToken": true,
